libs/platform/src/lib/components/search-field/search-field.component.ts

```diff
@@ -149,6 +149,12 @@ export class SearchFieldComponent extends BaseComponent implements OnInit, OnDes
      */
     @Input() isLoading = false;
 
+    /**
+     * ARIA label to specify what the search field is
+     * Not shown in the UI, only visible by the screen-readers.
+     */
+    @Input() ariaLabel: string;
```

Comment:
Let's add `aria-labelledby` option as well so it can be used with the table toolbar